.github/CONTRIBUTING.md

```diff
@@ -39,6 +39,82 @@ With that all in place, you're ready to start contributing to Carbon!
 
 ## Start contributing
 
+
+## Setting up the repository:
+## Windows guide:
+This repository runs best in a Linux envrionment, to avoid running into any errors as a windows user
+you will need to run the repo in a Linux environment using the Linux Subsystem for Windows.
+
+System requirements:
+https://docs.microsoft.com/en-us/windows/wsl/install-win10
+
+prerequisites:
+- aforementioned prerequisites
+- Windows Subsystem for Linux
+- linux operating system of your choice.
```

Comment:
```suggestion
- Linux operating system distribution of your choice.
```

Reply:
The Microsoft guide is not user friendly. Would you just link to the git docs to get a user started working in the repo?

Reply:
I wasn't suggesting to replace all of instructions with a link to the official WSL installation instructions, but I think we can replace the WSL installation portion with the link. I think it's reasonable to link the git docs to a user who is getting started with installing git. does that make more sense?

Reply:
Yeah if you think that would work better. Ideally the additions to the guide would be consistent with the current. Maybe including the link along with instructions would work better?

Reply:
Yeah if you think that would work better. Ideally the additions to the guide would be consistent with the current. Maybe including the link along with instructions would work better?